lib/cache/cache_test.go

```diff
@@ -267,6 +265,50 @@ func (s *CacheSuite) onlyRecentDisconnect(c *check.C) {
 	fixtures.DeepCompare(c, ca, out)
 }
 
+// TestWatchers tests watchers connected to the cache,
+// verifies that all watchers of the cache will be closed
+// if the underlying watcher to the target backend is closed
+func (s *CacheSuite) TestWatchers(c *check.C) {
+	p := s.newPackForAuth(c)
+	defer p.Close()
+
+	w, err := p.cache.NewWatcher(context.TODO(), services.Watch{Kinds: []services.WatchKind{
```

Comment:
```go
w, err := p.cache.NewWatcher(context.TODO(), services.Watch{Kinds: []services.WatchKind{
	services.WatchKind{
		Kind: services.KindCertAuthority,
	},
}})
```

Reply:
why extra boilerplate? Golan's latest addition makes it unnecessary